extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/config/HueBridgeConfig.java

```diff
@@ -36,6 +41,22 @@ public void setIpAddress(String ipAddress) {
         this.ipAddress = ipAddress;
     }
 
+    public int getPort() {
+        return HTTPS.equals(protocol) && port == 80 ? 443 : port;
```

Comment:
I would rather make `port` an `Integer` and return a default port if `null`, otherwise returning the custom port.

Reply:
Done.